pymc3/examples/SMC_2gaussians.py

```diff
@@ -0,0 +1,68 @@
+import pymc3 as pm
+import numpy as np
+from pymc3.step_methods import smc
+import theano.tensor as tt
+from matplotlib import pyplot as plt
+
+test_folder = ('ATMIP_TEST')
+
+n_chains = 300
+n_steps = 100
+tune_interval = 25
+n_jobs = 1
+
+n = 4
+
+mu1 = np.ones(n) * (1. / 2)
+mu2 = -mu1
+
+stdev = 0.1
+sigma = np.power(stdev, 2) * np.eye(n)
+isigma = np.linalg.inv(sigma)
+dsigma = np.linalg.det(sigma)
+
+w1 = stdev
+w2 = (1 - stdev)
+
+
+def last_sample(x):
+    return x[(n_steps - 1)::n_steps]
+
+
+def two_gaussians(x):
```

Comment:
Oh, I missed that. So mu1 and mu2 are not scalars? In that case it might be too much trouble. @AustinRochford Do you think that should be possible?